documentation/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Given this is the main entrypoint I wonder if we should drop the .sh from the script so that we're free later to reimplement in python if it gets complicated?